seqr/views/apis/anvil_workspace_api.py

```diff
@@ -236,7 +248,95 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
         project_name=project.name,
         sample_type=sample_type,
         genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+        dag_name = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=sample_type, version=DAG_VERSION),
         dag=json.dumps(pipeline_dag, indent=4),
```

Comment:
so the `pipeline_dag` variable is being constructed in this function, and is not the actual post body that the dag was triggered with. I think it would be better if we actually used the real post body in this message instead